doc/wiki/User-Guide/Projects/Subproject.md

```diff
@@ -197,6 +223,23 @@ The history contains all activities done directly refer to the current subprojec
 
 ![show subproject history](../../uploads/Screenshots/view_subproject_history.jpg)
 
+## Filter the history of a subproject
+
+**Description:**
+
+The entries in the history can be filtered by 4 different values: publisher, type of event, a start date and a end date, where a timeframe can be determined. When clicking o the search button, all values that are specified are used in the filter request.
```

Comment:
When clicking o the search button -> by clicking the search button

Reply:
done